docs/unified/clevo/post-install.md

```diff
@@ -122,3 +122,30 @@ power saving tweaks to enable sleep mode while a SATA disk is installed.
 
 1. Download the script: [link](https://raw.githubusercontent.com/Dasharo/dasharo-tools/main/clevo/sata-suspend-fixup.bat)
 2. Double click on the script to install the tweak
+
+### Headset jack fix for NV4x 12th Gen
+
+The headset jack in NV4x 12th Gen (Alder Lake) needs a fix that is available
+starting with kernel version v6.0. If you are using an older kernel, you need
+to add the fix manually:
+
+1. Create a file `/lib/firmware/hda-init.fw` with the following contents:
+
+    ```bash
```

Comment:
Don't we prefer using scripts https://github.com/Dasharo/dasharo-tools/tree/main/clevo to create these automatically to simplify post-install steps?
We can imagine we will gather more and more scripts like this and provide a single
post-install script in the documentation, which will detect what fixups are needed
on the given OS and given kernel/device/firmware/etc.

Reply:
Changed the doc to use the script instead of creating files manually.